inference-engine/src/low_precision_transformations/src/network_helper.cpp

```diff
@@ -1456,7 +1456,7 @@ size_t NetworkHelper::getParentOutputIndex(const std::shared_ptr<ngraph::Node>&
 std::vector<Output<Node>> NetworkHelper::getInputs(const std::shared_ptr<ngraph::Node>& node) {
     std::vector<Output<Node>> inputs(node->get_input_size());
     for (size_t i = 0; i < node->get_input_size(); ++i) {
-        inputs[i] = node->get_input_node_shared_ptr(i);
+        inputs[i] = node->get_input_source_output(i);
     }
     return inputs;
 }
```

Comment:
getInputs - is useless.

@eshoguli @itikhono please use `node->input_values()` which return exactly the same vector with outputs.

Reply:
Good point, thanks! Will fix it